sdk/evaluation/azure-ai-evaluation/CHANGELOG.md

```diff
@@ -55,6 +55,26 @@ evaluate(
 )
 ```
 
+- Simulator now requires configuration to call the prompty within the `azure_ai_project`. This enables key-less usage of the simulator.
```

Comment:
`AzureAIProject` and the model configs are separate for evaluators: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_model_configurations.py. I think we should keep them separate for simulator as well and not have two different schemas depending on what part of the SDK is being used.

Reply:
changed to use `model_config`